File: cs/Iris/ParserUtils/LexParseException/UndefinedException.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace Iris.ParserUtils.LexParseException
{
    public class UndefinedException : BaseException
    {
        public UndefinedException(int lNum, int cNum)
            : base(lNum, cNum)
        {

        }

        public override string ExceptionToString()
        {
            return base.stringTrace();
        }
    }
}

File: java/Iris/src/com/rokuan/iris/exception/UndefinedException.java
package com.rokuan.iris.exception;

public class UndefinedException extends BaseException
{
	public UndefinedException(int lNum, int cNum)
	{
		super(lNum, cNum);
	}

	public String ExceptionToString()
	{
		return super.stringTrace();
	}
}
